src/main.js

```diff
@@ -11,11 +11,12 @@ let appendableElement = null;
 function prepareDOM() {
 	const maganeContainer = document.createElement('div');
 	maganeContainer.id = 'maganeContainer';
-  appendableElement.insertAdjacentElement('afterbegin', maganeContainer);
+	appendableElement.insertAdjacentElement('afterbegin', maganeContainer);
 
+	// eslint-disable-next-line no-new
 	new Vue({
 		el: '#maganeContainer',
-		render: h => h(App)
+		render: ce => ce(App)
```

Comment:
This was because of an ESLint's `id-length` rule being set to `2`. Not sure if it was default or from `aqua` config, but I was sure I got everything installed matching `yarn.lock` file. The same applies for the `no-new` above.